estate/models/estate_propertis.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, fields
+
+class RealEstate(models.Model):
+    _name = "estate.propertis"
+    _description = "Test Model"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availibility = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        selection=[
+            ("north","North"), 
+            ("east","East" ),
+            ("west", "West") , 
+            ("south","South")
```

Comment:
```suggestion
            ("north", "North"), 
            ("east", "East" ),
            ("west", "West") , 
            ("south", "South")
```
maintain consistency in your code formatting.